cfgov/jinja2/v1/offices/_single.html

```diff
@@ -147,17 +147,20 @@ <h3 class="h2 qa-resource-title">
         {{ office.content | safe }}
     </section>
     {% endif %}
-    <aside>
+    <aside class="block
```

Comment:
I've seen us go back and forth on whether we want more classes in our markup or a smaller HTML file. I'm okay with either, but would want to make sure we don't go and forth.